src/wipplpy/modules/config_reader.py

```diff
@@ -0,0 +1,89 @@
+"""
+Define routines for accessing and extracting information from locally saved
+config files.
+"""
+
+import configparser
+import logging
+import os
+
+from WiPPLPy.modules.generic_get_data import lazy_get
+
+
+class ConfigReader:
+    """
+    Access the local INI file containing relevant MDSplus labels and define
+    methods for referring to each label.
```

Comment:
This docstring should also briefly mention the attributes `BRB_remote_server` and whatnot as seen here [https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring](url)